src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -98,10 +98,6 @@ private void emitAnnotationValue(CodeWriter codeWriter, String whitespace, Strin
     codeWriter.emit(whitespace + "}");
   }
 
-  public static AnnotationSpec of(Type annotation) {
-    return builder(annotation).build();
-  }
-
   public static Builder builder(Type type) {
```

Comment:
Oh BTW, we're not consistent between static factories (here) and public constructor (e.g. `JavaFile` or `CodeBlock`) for builders.